### PR TITLE
AG guidelines fix links

### LINKED ISSUE #126: Undefined reference to chapter 1.6.5 in DS Guidelines
In subsection about commands in [DS Guidelines](tango-doc/source/development/device-api/ds-guideline/TangoDesignGuidelines-Revision-7.rst) there is reference to chapter 1.6.5

> State: enumeration, representing the different states described in chapter 1.6.5. 

I can't find such chapter, is it from old documentation?

Maybe it should point here http://tango-controls.readthedocs.io/en/latest/overview/SimplifiedTangoDatamodel.html?highlight=state

### PR DESCRIPTION
**Pull request by andygotz**
Fixed many errors in the Guidelines document including English, images, and links. I will stop working on this for now because it fixes #126. 

Still needs more working on to split into smaller sections, reviewing and guidelines for missing Tango 9 features like Pipes.

### CODE REVIEW

File: source/development/device-api/ds-guideline/media/.~lock.image7.emf# [hunk 1]
```diff
@@ -0,0 +1 @@
+,goetz,pcandy,10.09.2017 16:19,file:///home/goetz/.config/libreoffice/4;
```
Comment (Ingvord) [inline source/development/device-api/ds-guideline/media/.~lock.image7.emf#]: I guess this was added falsy

File: source/installation/binary_package.rst [hunk 2]
```diff
@@ -4,8 +4,10 @@
 Binary packages
 ---------------
 
+Binary packages for Tango are maintained to make it easier for developers to
+install the latest builds of Tango without have to compile them. 
 
-
+Here is a list of the available binary packages.
 
 `JTango <https://bintray.com/tango-controls/maven/JTango/9.3.3>`_ - library for device servers and clients in Java.
```
Comment (Ingvord) [inline source/installation/binary_package.rst]: This one must point to latest:

```
<a href='https://bintray.com/tango-controls/maven/JTango/_latestVersion'><img src='https://api.bintray.com/packages/tango-controls/maven/JTango/images/download.svg'></a>
```

Comment (piogor) [reply source/installation/binary_package.rst]: I will take care of this.

File: source/development/device-api/ds-guideline/media/.~lock.image7.emf# (hunk 1 above)
Comment (piogor) [reply source/development/device-api/ds-guideline/media/.~lock.image7.emf#]: I will remove the file.

**Comment by piogor** [comment]
@andygotz , shell I delete the branch?

**Pull request merged**